src/components/Ingredients.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { v4 as uuid } from 'uuid';
+import List from '@mui/material/List';
+import Ingredient from './Ingredient';
+
+function Ingredients(props) {
+  const { recipeId, ingredients, editIngredient, removeIngredient } = props;
+
+  return (
+    <List component='div' disablePadding>
+      {ingredients.map(ingredient => (
+        <Ingredient
+          key={uuid()}
```

Comment:
A suggestion, instead of uuid you could compose the component key as something like: `recipeId_{index}`. You can get the index in the `map` function with something like:
```jsx
{ingredients.map((ingredient, index) => (
    <Ingredient
        key={`recipeId_${index}`}
        ...
    )
}
```

Reply:
Or use `ingredient.id`, probably simpler

Reply:
Good suggestion, applying it now